test/common/testlib.py

```diff
@@ -1229,61 +1228,6 @@ def assert_no_oops(self):
                 assert not self.is_visible("#navbar-oops"), "Cockpit shows an Oops"
 
 
-class _DebugOutcome(unittest.case._Outcome):  # type: ignore[name-defined]
```

Comment:
Dropping this would be amazing it hurt us once already with Python 3.11

Reply:
I tested this, and it seems to work fine now with python 3.11 🎉 Of course tearDown() was the obvious thing to consider back then, but apparently the default run() function worked differently back then, so that parts of the cleanup already happened. I'm glad this can become simpler now.

This might regress the artifacts that we get from running tests on RHEL 8, but I think this is acceptable.